src/components/card-grid.jsx

```diff
@@ -25,25 +26,30 @@ export default function CardGrid(props) {
       : `${process.env.GATSBY_BASE_IMAGE_URL}${info.image}`;
 
   return info ? (
-    <Grid item xs={12} sm={6} md={4}>
-      <Card className={classes.card}>
-        <CardMedia
-          className={classes.cardMedia}
-          image={defaultImage}
-          title={info.display}
-        />
-        <CardContent className={classes.cardContent}>
-          <Typography gutterBottom variant='h5' component='h2'>
-            {info.display}
-          </Typography>
-          <Typography>{post.excerpt}</Typography>
-        </CardContent>
-        <CardActions dir='rtl'>
-          <Button size='small' color='primary' href={slug.slice(0, -1)}>
-            <ArrowForwardIcon />
-          </Button>
-        </CardActions>
-      </Card>
+    <Grid item xs={props.xs} sm={props.sm} md={props.md}>
+      <CardActionArea
+        href={slug.slice(0, -1)}
+        style={{ textDecoration: 'none' }}
+      >
+        <Card className={classes.card}>
+          <CardMedia
+            className={classes.cardMedia}
+            image={defaultImage}
+            title={info.display}
+          />
+          <CardContent className={classes.cardContent}>
+            <Typography gutterBottom variant='h5' component='h2'>
+              {info.display}
+            </Typography>
+            <Typography>{post.excerpt.slice(0, 65)}</Typography>
```

Comment:
Why just 65 characters?

Reply:
If you just want 65 characters, you could use the gatsby graphql query, on the `pruneLength` change to 65
 ![image](https://user-images.githubusercontent.com/37890026/113328418-41455f00-92ea-11eb-83e8-cd4efd044598.png)